schematic/models/validate_attribute.py

```diff
@@ -1397,6 +1467,27 @@ def _gather_value_warnings_errors(
             )
         return errors, warnings
 
+    def _check_if_target_manifest_is_empty(
+        self,
+        target_manifest: pd.core.series.Series,
+        target_manifest_empty: list[bool],
+        column_names: dict[str, str],
+    ):
+        # Make a copy of the target manifest with only user uploaded columns
+        target_manifest_dupe = target_manifest.drop(
+            [column_names["component"], column_names["id"], column_names["entityid"]],
+            axis=1,
+        )
+
+        # Check if the entire target_manifest is empty
+        if sum(target_manifest_dupe.isnull().values.all(axis=0)) == len(
+            target_manifest_dupe.columns
+        ):
+            target_manifest_empty.append(True)
+        else:
+            target_manifest_empty.append(False)
+        return target_manifest_empty
```

Comment:
Why add the result to the list and not just return a bool?

Reply:
I'm not seeing the for loop...

Reply:
The function is called by `_run_validation_across_targets_set` which is called within a for loop in the function `_run_validation_across_target_manifests` at line 1808:

```
for target_manifest_id, target_dataset_id in zip(
            target_manifest_ids, target_dataset_ids
        ):
            # Pull manifest from Synapse
            entity = synStore.getDatasetManifest(
                datasetId=target_dataset_id, downloadFile=True
            )
            # Load manifest
            target_manifest = pd.read_csv(entity.path)

            # Get manifest column names
            column_names = self._get_column_names(target_manifest=target_manifest)

            # Read each target manifest and run validation of current manifest column (set) against each
            # manifest individually, gather results
            if "set" in rule_scope:
                (
                    (
                        missing_manifest_log,
                        present_manifest_log,
                        repeat_manifest_log,
                    ),
                    target_attribute_in_manifest_list,
                    target_manifest_empty,
                ) = self._run_validation_across_targets_set(
                    val_rule=val_rule,
                    column_names=column_names,
                    manifest_col=manifest_col,
                    target_attribute=target_attribute,

...etc...

```
Its basically attempting (like target_attribute_in_manifest_list) to save data for all the target manifests that are being evaluated.

Im doing them in slightly different ways bc I could reuse `_check_if_target_manifest_is_empty` for both value and set operations but i had to do it differently for set and value to get `target_attribute_in_manifest_list` bc of how the targets are processed differently.